streaming/src/main/scala/org/apache/spark/streaming/StreamingConf.scala

```diff
@@ -185,4 +185,10 @@ object StreamingConf {
       .longConf
       .createWithDefault(0)
 
+  private[streaming] val STREAMING_EXTRA_LISTENERS = ConfigBuilder("spark.streaming.extraListeners")
```

Comment:
If you don't mind, shall we match the code patter the other codes like line 183?

Reply:
Done